src/scanForInputs.js

```diff
@@ -6,9 +6,6 @@ const forms = new Map()
 
 // Accepts the DeviceInterface as an explicit dependency
 const scanForInputs = (DeviceInterface) => {
-    // Avoid autofill on our signup page
-    if (window.location.href.match(/^https:\/\/(.+\.)?duckduckgo\.com\/email\/choose-address/i)) return
```

Comment:
❓ Would we want the autofill to run for anything on `/email/`? 
And if not, would changing the regex to not be `choose-address` specific allow us to keep the check up here?
```js
if (window.location.href.match(/^https:\/\/(.+\.)?duckduckgo\.com\/email\/i)) return
```
I don’t really have any specific reason for wanting to keep the check here — though it does seem like the entry point that’s called from the device interfaces, which makes it a nice place to bail out.

Reply:
The old dashboard has a page where we show how autofill works within a field. It's true that normal users should never get there, but I'm pretty sure someone from the company or old testers will find out and think it's broken or that we will reinstate that functionality at some point.

I'm conflicted as well. There are plans to enable a more robust blocklist that works across platforms (currently only extensions have adopted it), but until then this is a quick fix that gets the job done 🤷‍♂️.